ceilometer_publisher_zeromq/queue.py

```diff
@@ -0,0 +1,55 @@
+from ceilometer import publisher
+from ceilometer.openstack.common.gettextutils import _
+from ceilometer.openstack.common import log
+from oslo.config import cfg
+import json
+import zmq
+
+LOG = log.getLogger(__name__)
+
+OPTS = [
+    cfg.StrOpt('publisher_zeromq_host',
+               default='127.0.0.1'),
```

Comment:
It supports the use of domain name lookups, perhaps localhost is not configured in /etc/hosts is the problem?